assets/pages/new-contributions-landing/components/ContributionThankYouContainer.jsx

```diff
@@ -2,38 +2,40 @@
 
 // ----- Imports ----- //
 
+import type { PaymentMethod } from 'helpers/contributions';
 import React from 'react';
 import { connect } from 'react-redux';
-
 import { type ThankYouPageStageMap, type ThankYouPageStage } from '../contributionsLandingReducer';
 import ContributionThankYou from './ContributionThankYou';
-import SetPassword from './SetPassword/SetPassword';
+import ContributionThankYouSetPassword from './ContributionThankYouSetPassword';
+import ContributionThankYouPasswordSet from './ContributionThankYouPasswordSet';
 
 // ----- Types ----- //
 
 /* eslint-disable react/no-unused-prop-types */
 type PropTypes = {
   thankYouPageStage: ThankYouPageStage,
+  paymentMethod: PaymentMethod,
 };
 /* eslint-enable react/no-unused-prop-types */
 
 const mapStateToProps = state => ({
   thankYouPageStage: state.page.form.thankYouPageStage,
+  paymentMethod: state.page.form.paymentMethod,
 });
 
-// TODO: change landing page copy if password has/hasn't been set
-const thankYouPageStage: ThankYouPageStageMap<React$Element<*>> = {
-  setPassword: (<SetPassword />),
-  thankYou: (<ContributionThankYou />),
-  thankYouPasswordSet: (<ContributionThankYou />),
-  thankYouPasswordNotSet: (<ContributionThankYou />),
-};
-
-
 // ----- Render ----- //
 
 function ContributionThankYouContainer(props: PropTypes) {
 
+  const thankYouPageStage: ThankYouPageStageMap<React$Element<*>> = {
+    thankYouSetPassword: (<ContributionThankYouSetPassword />),
+    thankYou: (<ContributionThankYou showDirectDebitCopy={props.paymentMethod === 'DirectDebit'} />),
+    thankYouPasswordDeclinedToSet: (<ContributionThankYou showDirectDebitCopy={false} />),
```

Comment:
Yes I agree it's not very clear. The reason is that if a Direct Debit user hits the thank you page having come from the setPassword screen then we don't want to show the Direct Debit copy that we have to show to be compliant. 

On reflection, I think it might be clearer to showDirectDebitCopy to default to false. But i'll also add a comment into `ContributionThankYou.jsx` explaining what situations we would expect to show the copy

Reply:
> The reason is that if a Direct Debit user hits the thank you page having come from the setPassword screen then we don't want to show the Direct Debit copy that we have to show to be compliant.

Is that because it's already been shown on the setPassword screen?

Reply:
Yes I agree it's not very clear. The reason is that if a Direct Debit user hits the thank you page having come from the setPassword screen then we don't want to show the Direct Debit copy that we have to show to be compliant. 

On reflection, I think it might be clearer to showDirectDebitCopy to default to false. But i'll also add a comment into `ContributionThankYou.jsx` explaining what situations we would expect to show the copy

Reply:
Yes, the setPassword screen is the first "thank you" page that they see for those who get offered the chance to set a password. We need to show the copy on the first page they see, which will be `setPassword` or `thankYou`

Reply:
I've added a comment now in `ContributionThankyou.jsx` in the `PropTypes` definition